content/chat/rooms/messages.textile

```diff
@@ -126,3 +146,235 @@ const MyComponent = () => {
   );
 };
 ```
+
+h2(#update). Update a message
+
+blang[javascript].
+  Use the "@messages.update()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_js.Messages.html#update method to update a message in a chat room. All users that are "subscribed":#subscribe to messages on that room will receive the update:
+
+blang[react].
+  Use the "@update()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_react.UseMessagesResponse.html#update method available from the response of the @useMessages@ hook to update a message in the room:
+
+```[javascript]
+import { Message } from '@ably/chat';
+const messageToUpdate: Message
+await room.messages.update(messageToUpdate, { "text": "my updated text" }, { description: 'Message update by user' });
+```
+
+```[react]
+import { useMessages } from '@ably/chat/react';
+import { Message } from '@ably/chat';
+
+const MyComponent = () => {
+  const { update } = useMessages();
+  const [message, setMessage] = useState<Message>();
+
+  const handleMessageUpdate = (msg: Message) => {
+  const newText = prompt('Enter new text');
+      if (!newText) {
+        return;
+      }
+    update(msg, { "text", newText }, { description: 'Message update by user' });
```

Comment:
```suggestion
  const handleMessageUpdate = async (msg: Message) => {
    await update(msg, { text: "my updated text" }, { description: "Message update by user" });
```
Suggestion:
- Make the function async and add an await, so that errors get logged. (Alternatively add a `.then` and `.catch`.)
- Remove the `prompt()` stuff because the JS example doesn't have that either.